src/composables/balances.ts

```diff
@@ -92,7 +95,15 @@ export function useBalances({ store }: IDefaultComposableOptions) {
       {},
     );
 
-    storeBalances(balances.value as Balances);
+    storeBalances(balances.value, activeNetwork.value.networkId);
+  }
+
+  if (!activeNetworkId) {
+    activeNetworkId = activeNetwork.value.networkId;
+  } else if (activeNetwork.value.networkId !== activeNetworkId) {
+    activeNetworkId = activeNetwork.value.networkId;
+    balances.value = getStoredBalances(activeNetworkId);
+    updateBalances();
```

Comment:
```suggestion
activeNetworkId = activeNetwork.value.networkId;
activeNetworkId !== activeNetworkId ? (balances.value = getStoredBalances(activeNetworkId), updateBalances()) : null;
```

Please see if this does the work

Reply:
`activeNetworkId !== activeNetworkId` - this is always false :)